server/command_test.go

```diff
@@ -321,21 +321,21 @@ func TestPlugin_ExecuteCommand_Installation(t *testing.T) {
 		"install inaccessible server instance": {
 			numInstances:      0,
 			commandArgs:       &model.CommandArgs{Command: "/jira install server https://inaccessible-jira-server.com", UserId: mockUserIDSysAdmin},
-			expectedMsgPrefix: "Jira server returned http status code \"404\" when checking for availability: \"https://inaccessible-jira-server.com\"",
+			expectedMsgPrefix: `it looks like we couldn't validate the connection to your Jira server. Please make sure the URL was entered correctly. This could also be because of existing firewall or proxy rules. If you intend to have a one way integration from Jira to Mattermost this is not an issue: Jira server returned http status code "404" when checking for availability: "https://inaccessible-jira-server.com"`,
 		},
 		"install valid server instance in wrong state": {
 			numInstances:      0,
 			commandArgs:       &model.CommandArgs{Command: "/jira install server https://starting-jira-server.com", UserId: mockUserIDSysAdmin},
-			expectedMsgPrefix: "Jira server is not in correct state, it should be up and running: \"https://starting-jira-server.com\"",
+			expectedMsgPrefix: `it looks like we couldn't validate the connection to your Jira server. Please make sure the URL was entered correctly. This could also be because of existing firewall or proxy rules. If you intend to have a one way integration from Jira to Mattermost this is not an issue: Jira server is not in correct state, it should be up and running: "https://starting-jira-server.com"`,
```

Comment:
```suggestion
			expectedMsgPrefix: `we couldn't validate the connection to your Jira server. This could be because of existing firewall or proxy rules, or because the URL was entered incorrectly. If you intend to have a one-way integration from Jira to Mattermost this is not an issue: Jira server is not in correct state, it should be up and running: "https://starting-jira-server.com"`,
```